spec/requests/shots_spec.rb

```diff
@@ -5,7 +5,7 @@
     let(:player_1)   { Player.new(Board.new(4),'kasjdfhlai4oeiufrwh')}
```

Comment:
I think we could set these up in the game factory bot rather than in this test file. small refactor!